packages/ppx/test/native/Static_test.re

```diff
@@ -311,6 +311,108 @@ let properties_static_css_tests = [
     [%expr [%css "background-color: red"]],
     [%expr CssJs.backgroundColor(CssJs.red)],
   ),
+  (
+    [%css "background-color: color-mix(in srgb, white 10%, red)"],
+    [%expr [%css "background-color: color-mix(in srgb, white 10%, red)"]],
+    [%expr
+      CssJs.backgroundColor(
+        `colorMix2((
+          (`in_, `srgb),
+          (CssJs.white, `percent(10.)),
+          (CssJs.red, `percent(90.)),
+        )),
+      )
+    ],
+  ),
+  (
+    [%css "background-color: color-mix(in srgb, white 10%, red 10%)"],
+    [%expr [%css "background-color: color-mix(in srgb, white 10%, red 10%)"]],
+    [%expr
+      CssJs.backgroundColor(
+        `colorMix2((
+          (`in_, `srgb),
+          (CssJs.white, `percent(0.5)),
+          (CssJs.red, `percent(0.5)),
+        )),
+      )
+    ],
+  ),
+  (
+    [%css "background-color: color-mix(in srgb, white, red)"],
+    [%expr [%css "background-color: color-mix(in srgb, white, red)"]],
+    [%expr
+      CssJs.backgroundColor(
+        `colorMix2((
+          (`in_, `srgb),
+          (CssJs.white, `percent(50.)),
+          (CssJs.red, `percent(50.)),
+        )),
+      )
+    ],
+  ),
+  (
+    [%css "background-color: color-mix(in srgb, white, red 10%)"],
+    [%expr [%css "background-color: color-mix(in srgb, white, red 10%)"]],
+    [%expr
+      CssJs.backgroundColor(
+        `colorMix2((
+          (`in_, `srgb),
+          (CssJs.white, `percent(90.)),
+          (CssJs.red, `percent(10.)),
+        )),
+      )
+    ],
+  ),
+  (
+    [%css "background-color: color-mix(in srgb, white 100%, red 100%)"],
+    [%expr
+      [%css "background-color: color-mix(in srgb, white 100%, red 100%)"]
+    ],
+    [%expr
+      CssJs.backgroundColor(
+        `colorMix2((
+          (`in_, `srgb),
+          (CssJs.white, `percent(0.5)),
+          (CssJs.red, `percent(0.5)),
+        )),
+      )
+    ],
+  ),
+  (
+    [%css "background-color: color-mix(in srgb-linear, white, red)"],
+    [%expr [%css "background-color: color-mix(in srgb-linear, white, red)"]],
+    [%expr
+      CssJs.backgroundColor(
+        `colorMix2((
```

Comment:
I know I proposed colorMix2/colorMix4, but in reality what makes a bit more sense is to keep colorMix as a single arity but then the first argument of the variant is `in1/`in2/`in3, does that make sense?

Reply:
You mean more or less like this?
```
 `colorMix((
     `in1(`in_, `srgb),
       (CssJs.white, `percent(10.)),
       (CssJs.red, `percent(90.)),
 ))
```

Reply:
if that's the case, `method<>` sounds better as it portrays the name of the argument

Reply:
More like:

```
 `colorMix((
     `in1(`srgb),
       (CssJs.white, `percent(10.)),
       (CssJs.red, `percent(90.)),
 ))
```

Reply:
Done!